app/grants/models.py

```diff
@@ -402,9 +395,11 @@ def do_execute_subscription_via_web3(self, minutes_to_confirm_within=5):
     def helper_tx_dict(self, minutes_to_confirm_within=5):
         """returns a dict like this: {'to': '0xd3cda913deb6f67967b99d67acdfa1712c293601', 'from': web3.eth.coinbase, 'value': 12345}"""
         from dashboard.utils import get_nonce
+        from dashboard.utils import get_web3
+        web3 = get_web3(self.grant.network)
         return {
             'from': settings.GRANTS_OWNER_ACCOUNT,
-            'nonce': get_nonce(self.grant.network, settings.GRANTS_OWNER_ACCOUNT),
+            'nonce': web3.eth.getTransactionCount(settings.GRANTS_OWNER_ACCOUNT),
```

Comment:
why was this changed?  this will cause any pending transactions to pile on top of each other at the same nonce.

Reply:
`get_nonce()` handles this scenario successfully

Reply:
Any attempted transactions that didnt reach the chain increased the nonce in the DB which caused all transactions for the account to fail. I can change it back since the subminer is reaching the chain now and DB should be able to stay in sync.

Reply:
>  I can change it back since the subminer is reaching the chain now and DB should be able to stay in sync.

makes sense to me